lib/body-parser.js

```diff
@@ -3,20 +3,16 @@ module.exports = function codeBodyParser() {
 	return function bodyParser(req, res, next) {
 		
 		let body = '';
-		req.on('data', chunk => {
-			body += chunk;
-		});
 
+		req.on('data', chunk => body += chunk);
+		req.on('error', err => next(err));
 		req.on('end', () => {
-			if (body !== '') {
+			if (req._body !== '') { 
```

Comment:
`req._body` property never exists on the `req` object. You actually want to refer to your `body` variable.